src/vm/gcenv.ee.cpp

```diff
@@ -843,6 +843,8 @@ void GCToEEInterface::DiagWalkBGCSurvivors(void* gcContext)
 
 void GCToEEInterface::StompWriteBarrier(WriteBarrierParameters* args)
 {
+    int flushXrestart = SWB_PASS;
```

Comment:
>flushXrestart [](start = 8, length = 13)

I don't understand the 'X' here in the name.... does it mean and/or? :P perhaps rename it to something like stompWBCompleteActions?

Reply:
Well thats my dumb style for describing possible flags in enum variable. Will change to `stompWBCompleteActions`

Reply:
usually I don't point out styling related things 'cause people have different preferences but I thought this name was a bit difficult to decipher. thanks for making the changes!